pkg/api/message/v3/service.go

```diff
@@ -16,21 +17,23 @@ import (
 type Service struct {
 	proto.UnimplementedMlsApiServer
 
-	log          *zap.Logger
-	waku         *wakunode.WakuNode
-	messageStore *store.Store
-	mlsStore     mlsstore.MlsStore
+	log               *zap.Logger
+	waku              *wakunode.WakuNode
+	messageStore      *store.Store
+	MLSStore          mlsstore.MlsStore
```

Comment:
Does this one need to be public or did it get caught up in the find/replace?

Reply:
Good call. Totally got caught up in the find or replace.